docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -0,0 +1,423 @@
+# Prerequisite
+
+## Possible fields
+- `parent_id`
+- `name`
+- `opening_status`
+- `org_unit_type_id`
+- `groups`
+- `geo_point`
+- `shape`
+- `instances`
+
+# POST /api/orgunits/changes/?app_id=..
+
+API used to create or modify an OrgUnit with validation (see feature flags)
+When creating an OrgUnit, the client doesn't know the ID or the OrgUnit was not created yet and we might want to apply 
+some changes still. Therefore, the IDs can be either the DB id or the UUID generated by the client.
+Changes that are not provided are considered not being part of the change request.
+
+## Permissions
+
+User must be authenticated.
+
+## Query parameters
+
+- app_id: String - Must be provided, project for which this is created.
+
+## Body
+
+```json
+{
+  "uuid": "UUID - Client generated UUID",
+  "org_unit_id": "String - id or UUID of the OrgUnit to change",
+  "created_at": "Timestamp - format same as /api/instances",
+  "updated_at": "Timestamp - format same as /api/instances",
+  "changes": { "": "Changes is not null and must contain at least one field",
+    "parent_id": "String? - id or UUID of the parent OrgUnit, may be null or omitted.",
+    "name": "String? - Name of the OrgUnit, may be null or omitted.",
+    "opening_status": "Enum<Status>? - One of `open` or `close`",
+    "org_unit_type_id": "Int? - id of the OrgUnitType, may be null or omitted",
+    "groups": "Array of Group ids? - can be empty, null or omitted. Empty means we want to remove all values",
+    "geo_point": { "": "New geopoint for the OrgUnit, may be null or omitted",
+      "latitude": "Double - New latitude of the OrgUnit",
+      "longitude": "Double - New longitude of the OrgUnit",
+      "altitude": "Double - New altitude of the OrgUnit",
+      "accuracy": "Double - New accuracy of the OrgUnit"
+    },
+    "geo-shape": "String? - GeoJSON representation, may be null or omitted",
+    "instances":  "Array of instance ids? - may be null or omitted, cannot be empty"
```

Comment:
```suggestion
  "uuid": "UUID - Client generated UUID",
  "org_unit_id": "String - id or UUID of the OrgUnit to change",
  "created_at": "Timestamp - format same as /api/instances",
  "updated_at": "Timestamp - format same as /api/instances",
  "new_parent_id": "String? - id or UUID of the parent OrgUnit, may be null or omitted.",
  "new_name": "String? - Name of the OrgUnit, may be null or omitted.",
  "new_org_unit_type_id": "Int? - id of the OrgUnitType, may be null or omitted",
  "new_groups": "Array of Group ids? - can be empty, null or omitted. Empty means we want to remove all values",
  "new_geo_point": { "": "New geopoint for the OrgUnit, may be null or omitted",
    "latitude": "Double - New latitude of the OrgUnit",
    "longitude": "Double - New longitude of the OrgUnit",
    "altitude": "Double - New altitude of the OrgUnit",
    "accuracy": "Double - New accuracy of the OrgUnit",
  },
  "new_reference_instances":  "Array of instance ids? - may be null or omitted, cannot be empty"
```

As discussed this morning, I suggest using a flat structure in the returned payload.

It makes the code clearer and more succinct on the back-end side (more meaningful variable names and no need of nested Serializers).

> [Flat is better than nested.](https://peps.python.org/pep-0020/) 🤣